lib/cisco_node_utils/cisco_cmn_utils.rb

```diff
@@ -166,6 +166,53 @@ def self.zero_pad_macaddr(mac)
       sprintf('%04x.%04x.%04x', o1, o2, o3)
     end
 
+    # This will expand the string to a list of bds as integers
+    def self.string_to_array(string)
```

Comment:
Okay, but they are very specific to bridge_domain_vni so please rename them to reflect that; i.e. `Utils.string_to_array` is pretty vague for a common utility method. Also, they should not be in this file unless you expect they'll be needed in more than one provider.